lib/generators/geo_concerns/install_generator.rb

```diff
@@ -46,7 +46,14 @@ def install_image_work
 
     def install_file_sets_controller
       file_path = 'app/controllers/curation_concerns/file_sets_controller.rb'
-      copy_file 'controllers/curation_concerns/file_sets_controller.rb', file_path
+      if File.exist?(file_path)
+        inject_into_file file_path, after: /include CurationConcerns::FileSetsControllerBehavior.*$/ do
+          "\ninclude GeoConcerns::FileSetsControllerBehavior" \
+            "\ninclude GeoConcerns::MessengerBehavior\n"
```

Comment:
Can the indent be consistent? Also looking at the [controller template](https://github.com/projecthydra-labs/geo_concerns/blob/48b9120a92a798d7699cea6783eaf930d0789367/lib/generators/geo_concerns/templates/controllers/curation_concerns/file_sets_controller.rb), there should probably be 4 spaces between the newline and the include in both lines 51 and 52.